app/lib/submission_builder/ty2024/states/md/documents/md502b.rb

```diff
@@ -19,7 +19,7 @@ def document
                     xml.Dependent do
                       xml.Name do
                         xml.FirstName sanitize_for_xml(dependent.first_name, 16)
-                        xml.MiddleInitial sanitize_for_xml(dependent.middle_initial, 1) if dependent.middle_initial.present?
+                        xml.MiddleInitial sanitize_middle_initial(dependent.middle_initial) if sanitize_middle_initial(dependent.middle_initial).present?
```

Comment:
lines 185 and 189 also have `xml.MiddleInitial`

Reply:
> I did not sanitize account_holder_middle_initial b/c this is input through intake & we validate this field.

Reply:
sorry for missing the coment!